sshd-core/src/main/java/org/apache/sshd/common/channel/ChannelOutputStream.java

```diff
@@ -195,9 +195,7 @@ public synchronized void write(byte[] buf, int s, int l) throws IOException {
     public synchronized void flush() throws IOException {
         Channel channel = getChannel();
         if (!isOpen()) {
```

Comment:
I don't think this is a good idea - what if the user meant to flush the stream but did not notice it is closed. The code would simply "swallow" the exception. AFAIK the stream contract is clear: do not call **any** method after calling *close()* (one *can* call *close()* as many times as one wants since it is idempotent),